tests/acceptance/01_vars/01_basic/this_variables.cf

```diff
@@ -0,0 +1,101 @@
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+bundle agent init
+{
+  files:
+
+      "$(G.testfile).expected"
+      copy_from => local_cp("$(this.promise_filename).finish");
+
+}
+
+bundle agent test
+{
+  vars:
+    "var"      slist => { "var in test" };
+    "var_test" slist => { "var_test in test" };
+
+  files:
+    "$(G.testfile).actual"
+    create => "true",
+    edit_line => test_edit_line("testline");
+
+  methods: 
+    "any" usebundle => inter;
```

Comment:
what does this have and the bundles `inter` and `final` have to do with the edit_lines test?

Reply:
Wait a sec.
- Init (copy G.testfile.expected from .finish
- Test (Promise to create and edit G.testfile.actual with edit_line)
- check if .actual (the one we edited) is different from .expected (which is a copy of .finish)

This looks like a broken test to me. The .actual file was not seeded with any content to begin with, and none of the edit_line promises insert `testline`. That is why `testline` is missing from the end state. Perhaps there was an intention to seed the .actual file with some content  that already included `testline` similar to how .expected was seeded from .finish.